src/overlays/actors/ovl_Bg_Hidan_Curtain/z_bg_hidan_curtain.c

```diff
@@ -86,11 +86,12 @@ void BgHidanCurtain_Init(Actor* thisx, PlayState* play) {
     this->treasureFlag = (thisx->params >> 6) & 0x3F;
     thisx->params &= 0x3F;
 
-    if ((this->actor.params < 0) || (this->actor.params > 0x3F)) {
+    if (OOT_DEBUG && ((this->actor.params < 0) || (this->actor.params > 0x3F))) {
```

Comment:
can this not be `#if OOT_DEBUG` ? or are we preferring `if (OOT_DEBUG)` over using the preprocessor (idk)

(personally would prefer `#if` when it works, it's more obvious imo)

Reply:
This can definitely be `#if OOT_DEBUG`, so we have a styling choice here.